GliaWidgets/Component/Button/Style/Properties/VisitorCodeCloseButtonProperties.swift

```diff
@@ -0,0 +1,8 @@
+import UIKit
+
+struct VisitorCodeCloseButtonProperties: ButtonProperties {
+    let image: UIImage? = Asset.close.image
+    let width: CGFloat? = 12
+    let height: CGFloat? = 12
+    let touchAreaInsets: TouchAreaInsets? = (dx: -10.0, dy: -10.0)
```

Comment:
Where is this used? I am very curious because I didn't know that this was possible :)